### ISSUE TITLE
_init() not work in visual script

### ISSUE THREAD

**Issue by jihot2000** [godotengine/godot#32765]
**Godot version:**  Godot 3.2 alpha2

**OS/device including version:** Ubuntu 18.04 LTS x86_64

**Issue description:** _init() not work in visual script

**Steps to reproduce:**
Create a simple visual script like:
![init_bug_screenshot](https://user-images.githubusercontent.com/5593546/66697043-7b8cfc00-ed04-11e9-9f73-55258f2c3742.png)

Nothing was printed in the output tab when running.

I also tried to assign a value to a member variable in _init(). But I found its value was not changed when I printed in _ready() function.

**Minimal reproduction project:**

**Comment by fire** [comment]
Can you use _ready() for your purposes?

**Comment by jihot2000** [comment]
> Can you use _ready() for your purposes?

Yes. I've moved the init logic to _ready(). It took me some time to find this issue. I think the _init() bug should be fixed for completeness of visual script.

**Comment by KoBeWi** [comment]
Still valid in 207cfc9
